### DISCUSSION TITLE
Deleting Chatflow

### DISCUSSION THREAD

**Discussion by koayst** [FlowiseAI/Flowise#2100] [Q&A]
I noticed that if I didn't delete the Chatflows and uninstall flowise (npm uninstall -g flowise), the same Chatflows which were not deleted before the uninstallation will be reinstated after I re-install flowise.

**Comment by Jaredude** [comment]
This would be the expected behavior. The flows are stored in the /data directory (or a DB) depending on your ENV variables supplied.
